### DISCUSSION TITLE
希望能在个人空间动态页实现禁止跳转动态详情

### DISCUSSION THREAD

**Discussion by AzusaHana** [the1812/Bilibili-Evolved#3265] [功能建议 / Ideas]
在个人空间的动态页，比如https://space.bilibili.com/1531395755/dynamic
点击动态文字会跳转动态详情页，希望能像动态首页那样，禁止动态点击后跳转详情页, 方便选择其中的文字。

**Comment by Geeyun-JY3** [comment]
最近动态相关功能失效了：#3191

**Reply by AzusaHana** [reply]
动态首页这个功能是正常的，个人空间动态页会跳转

**Reply by Geeyun-JY3** [reply]
那是因为 [2.1.8](https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.8) 修了。

而且你这算 bug，应该开 issue，我觉得既然都是动态相关功能，可以一起算进 #3191。

**Reply by the1812** [reply]
本来也是支持的, 算是 #3191 的一个附带问题, 需要有一个更好的区分新版动态的方法, 用 cookie 的话在个人空间也会判断为新版所以失效了
